pkg/kubelet/cm/event_dispatcher.go

```diff
@@ -106,6 +127,19 @@ func (ed *eventDispatcher) dispatchEvent(ev *lifecycle.Event) (*lifecycle.EventR
 	return mergedReplies, utilerrors.NewAggregate(errlist)
 }
 
+func (ed *eventDispatcher) updateIsolators() {
+	isolators := make([]string, len(ed.isolators))
+	idx := 0
+	for isolator := range ed.isolators {
```

Comment:
output from `MapKeys()` is `[]reflect.Value`. You cannot cast it directly to `[]string` so you need to create another loop. This loop will iterate through each element of `[]reflect.Value`, cast it to `string` and append to `[]string`.